src/zenml/integrations/kubeflow/flavors/kubeflow_orchestrator_flavor.py

```diff
@@ -41,6 +42,8 @@ class KubeflowOrchestratorSettings(BaseSettings):
             specified on steps.
         timeout: How many seconds to wait for synchronous runs.
         client_args: Arguments to pass when initializing the KFP client.
+        client_username: Username to use for the kubeflow client. To be used together with `client_password`.
```

Comment:
I think this should mention that we're trying to generate the cookie from those to (and that they both need to be set for this to work).